src/project-management/android-app.ts

```diff
@@ -135,7 +179,21 @@ export class AndroidApp {
   }
 }
 
+/**
+ * A SHA-1 or SHA-256 certificate.
+ *
+ * Do not call this constructor directly. Instead, use
+ * [`projectManagement.shaCertificate()`](admin.projectManagement.ProjectManagement#shaCertificate).
+ */
 export class ShaCertificate {
+  /**
```

Comment:
Documentation here deviates from the `d.ts` file:

https://github.com/firebase/firebase-admin-node/blob/646d90ff26bd4df6e53bf2f8a2d362ed7e4a537a/src/project-management.d.ts#L23-L43